scripts/create_backup_pd

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
```

Comment:
PLease add errorr handling or use set -ex. The way it is right now, some step might fail and user will not be aware of it.